judging/scripts/timeout_killer.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License.
+
+# Enforces the timeout for generating prescriptions.
+# After the designated amount of time has elapsed, kills the process with the PID provided in $1 and any
+# child processes thereof.
+#
+# The killing is done abruptly with SIGKILL so the output from the terminated process will be in an indeterminate
+# state.
+#
+# Information about the kill is written to stderr (and will be picked up in the log file of the main process, assuming
+# stderr is redirected.)
+
+# See https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html for what these do
+set -o errexit
+set -o errtrace
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+pid=$1
+
+# Allow this amount of time before killing the job
+KILL_AFTER=6.5h
+
+# Wait until designated time has elapsed
+sleep $KILL_AFTER
```

Comment:
I checked around StackOverflow etc. and there were a couple of options that floated to the top:
1) Use `timelimit` -- an Ubuntu package that has to be installed
2) Just sleep and then kill

I'm using option (2).

So this is ultra simple. Just sleep for 6½ hours, then kill the main job if it's running. That's it.

Reply:
That should do the job!